public/app/screens/main/sections/device-list/devices.jsx

```diff
@@ -29,7 +29,7 @@ var DevicesSection = React.createClass({
 
     return (
       <div className='cc-devices grid-14 padded'>
-        <DeviceFinder />
+        <DeviceFinder className='cc-devices__device-finder' />
```

Comment:
Why don't you just add the styles to the `DeviceFinder` or create a *container* for that?